app/scripts/controllers/main.js

```diff
@@ -7,11 +7,75 @@
  * # MainCtrl
  * Controller of the todoApp
  */
+//Initializing AngularJS app by creating a module, app name is 'todoApp'
+
+ //Creating a controller named 'MainCtrl' and adding it to our app
+ //adding '$scope' into the function so that our view has access to the controller's
+ //data. This is called dependency injection
+
 angular.module('todoApp')
-  .controller('MainCtrl', function () {
-    this.awesomeThings = [
-      'HTML5 Boilerplate',
-      'AngularJS',
-      'Karma'
-    ];
-  });
+  .controller('MainCtrl', function ($scope, $window, Task, ngNotify) {
+
+    //Getting user token and adding it to scope
+    $scope.userToken = localStorage.getItem("token");
```

Comment:
Same as the comment on standardized token variable naming that I made on authCtrl.